src/actions/authActions.js

```diff
@@ -0,0 +1,8 @@
+import { TEST_DISPATCH } from './types';
+
+export const loginUser = userData => {
```

Comment:
I think it will be better if you can dispatch the action instead of returning it

Reply:
okay, thanks.